RecoTauTag/HLTProducers/src/HLTTauIPFilter.cc

```diff
@@ -0,0 +1,94 @@
+/** \class HLTTauIPFilter
```

Comment:
```suggestion
/** \class HLTPFTauTIPFilter
```
To follow the names of the DataFormats more closely, I would suggest to use `HLTPFTauTIPFilter`.

Reply:
actually I chose IP instead of TIP because despite the "PFTauTransverseImpactParameter" name of that class, you can also retrieve the 3D impact parameter [1] and cut on it. This is why I did not the "T" (transverse) in the filter name. 
I don't have strong opinions on this filter name though.

[1] https://github.com/cms-sw/cmssw/blob/master/RecoTauTag/RecoTau/plugins/PFTauTransverseImpactParameters.cc#L144

Reply:
That's okay with me (too bad the name of the DataFormat is a bit misleading).

Reply:
Should have been clearer: no problem with "IP", but I would still prefer "PFTau".
Could we please use `HLTPFTauIPFilter` since the filter deals with "PFTau"s? (atm, we have other filters/producers in the menu named `HLTPFTau*`, but I couldn't find any called `HLTTau*`)